<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Controller</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QProgressBar" name="progressBarX">
      <property name="minimum">
       <number>-1024</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QProgressBar" name="progressBarZ">
      <property name="minimum">
       <number>-1024</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QProgressBar" name="progressBarY">
      <property name="minimum">
       <number>-1024</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QProgressBar" name="progressBarPhi">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%v</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Magnet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
